107/src/test/resources/ehcache-107-default-copiers.xml

```diff
@@ -35,4 +40,29 @@
     <ehcache:heap>10</ehcache:heap>
   </ehcache:cache-template>
 
+  <ehcache:cache-template name="stringtemplate">
```

Comment:
Why reuse an existing XML instead of creating a new one? Separation of concerns.